Removes a diagnostic message added for use by GTK developers.  This message can
cause significant performance issues with Flash- or javascript-heavy applications.

Issue: https://bugzilla.gnome.org/show_bug.cgi?id=590690
Patch: http://bbs.archlinux.org/viewtopic.php?id=76527

--- gtk+-2.16.5/gdk/x11/gdkxid.c        2009-06-19 04:59:18.000000000 +0200
+++ gtk+-2.16.5/gdk/x11/gdkxid.c.new    2009-07-22 11:30:12.000000000 +0200
@@ -56,10 +56,10 @@
   if (!display_x11->xid_ht)
     display_x11->xid_ht = g_hash_table_new ((GHashFunc) gdk_xid_hash,
 					    (GEqualFunc) gdk_xid_equal);
-
+/*
   if (g_hash_table_lookup (display_x11->xid_ht, xid))
     g_warning ("XID collision, trouble ahead");
-
+*/
   g_hash_table_insert (display_x11->xid_ht, xid, data);
 }
